presto-orc/src/test/java/com/facebook/presto/orc/metadata/TestDwrfMetadataReader.java

```diff
@@ -151,6 +155,24 @@ public void testReadFooter()
         assertEquals(footer.getDwrfStripeCacheOffsets().get(), stripeCacheOffsets);
     }
 
+    @Test
+    public void readStripeFooter_throwsForLargeStreams()
```

Comment:
our style is to use pure camel case and not have underscores in method names, and also our test names usually begin with "test", so the name should be

testReadStripeFooterThrowsForLargeStreams

Reply:
Updated, thanks. Also updated the other cranky names in this test.